app/views/devise/registrations/new.haml

```diff
@@ -0,0 +1,31 @@
+%body
```

Comment:
I generated the Devise views so I could style a few things. Unfortunately...that results in a much larger PR diff 🙈